src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -149,7 +149,7 @@ public ProcessResult updateCase(
                 );
 
                 if (errorMsg.isPresent()) {
-                    return new ProcessResult(singletonList(errorMsg.get()), emptyList());
+                    return new ProcessResult(emptyList(), singletonList(errorMsg.get()));
```

Comment:
I think this whole bit is wrong - updateCaseInCcd should throw an exception instead of returning this optional. We're supposed to fail for a generic reason and not pass details of what went wrong in CCD to the caseworker.

Reply:
I think you are right there is no point of showing a 422 error to caseworker and also this 422 won't be fixed on retrying so yes we should throw exception which would result into generic error but also we should log it so that we would be alerted.

Reply:
for starters #1035 does throw exception on method call `updateCaseInCcd` :wink: . Of course, will need further review after that but if it can be unified/simplified on that bit - it should go on and do that first thing